tests/Parser/ParserAbstractTest.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Findologic\PlentymarketsTest\Parser;
+
+use Findologic\Plentymarkets\Exception\InvalidUrlException;
+use Findologic\Plentymarkets\Parser\ParserAbstract;
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+
+class ParserAbstractTest extends TestCase
+{
+    public function correctDomainUrlProvider(): array
+    {
+        return [
+            ['http:///example.com', false],
```

Comment:
Split this up into two tests (two data providers). A test should only test the same output. In this case we are ensuring an exception is thrown, which is different then asserting that the domain was parsed correctly.